webapp/src/components/BuyParcelByMortgagePage/MortgageForm/MortgageForm.js

```diff
@@ -288,33 +285,18 @@ export default class MortgageForm extends React.PureComponent {
             </Grid.Column>
           </Grid.Row>
         </Grid>
-        <TxStatus.Idle isIdle={isTxIdle} />
-        {isPending ? (
-          <Message icon>
-            <Icon name="circle notched" loading />
-            <Message.Content>
-              <TxStatus.Text
-                txHash={publication.tx_hash}
-                txStatus={publication.tx_status}
-              />
-            </Message.Content>
-          </Message>
-        ) : null}
```

Comment:
Why are we deleting the pending Message here? I'm not necessarily opposed but I think it can be useful.

Also, upon further instruction, `isIdle` marks if the transaction started but hasn't been confirmed by the user. For example if you leave the metamask popup open. It's not that important, but might be useful for disallowing new txs until the last one is confirmed